TESTS/integration/README.md

```diff
@@ -0,0 +1,72 @@
+# Integration tests for Mbed OS
+
+
+
+
+Mbed OS provides a set of integration tests to help users identify integration issues around storage device and connectivity device.
+
+## Test suites
+
+| **Test suite** | **Description** |
+| ------------- | ------------- |
+| `fs-single` | File system single-threaded tests with write buffer sizes - 1 byte, 4b, 16b, 64b, 256b, 1kb, 4kb, 16kb, 64kb. |
+| `fs-multi` | File system multithreaded test with write buffer sizes - 1 byte, 4b, 16b, 64b, 256b, 1kb, 4kb, 16kb, 64kb. |
+| `net-single` | Network single-threaded test with receive buffer sizes - 128 bytes, 256b, 1kb, 2kb, 4kb. |
+| `net-multi` | Network multithreaded test for 1, 2 and 3 download threads with 1kb receive buffer size. |
+| `stress-net-fs` | Network and file system single and multithreaded tests:<ul><li>memory allocation - 10k, 20k, 40k, 60k</li><li>1 thread (sequential) - 1 download (1kb buffer), 1 file thread (1kb buffer)</li><li>2 parallel threads - 1 download, 1 file thread (1kb buffer)</li><li>3 parallel threads - 1 download, 2 file (256 bytes, 1 kb buffer)</li><li>4 parallel threads - 1 download, 3 file (1 byte, 256 bytes, 1kb buffer)</li></ul> |
+
+
+
+## Execute tests
+
+The integration test requires both the connectivity and and storage device to to configured. 
+If a platform have both of them enabled by default. The tests can be compiled and run the tests inside mbed-os directory, the configuration file need to be passed in the command line: 
```

Comment:
No need to use a configuration file when using the default configuration.
I'd suggest rewording to something like this:
"If a platform has both of them enabled by default (in `targets.json`), then the tests can be compiled and run from inside the mbed-os directory:"